metadeploy/api/serializers.py

```diff
@@ -502,16 +512,19 @@ def validate(self, data):
                 )
             )
 
-        user_has_valid_token = all(user.token)
-        if not user_has_valid_token:
-            raise serializers.ValidationError(
-                _("The connection to your org has been lost. Please log in again.")
-            )
+        # @@@ TODO: What to do with this in the case of a Scratch Org?
+        if user:
+            user_has_valid_token = all(user.token)
+            if not user_has_valid_token:
+                raise serializers.ValidationError(
+                    _("The connection to your org has been lost. Please log in again.")
+                )
 
-        data["org_name"] = user.org_name
-        data["org_type"] = user.org_type
-        data["full_org_type"] = user.full_org_type
-        data["organization_url"] = user.instance_url
+        # @@@ TODO: What to do with this in the case of a Scratch Org?
+        data["org_name"] = user.org_name if user else None
+        data["org_type"] = user.org_type if user else None
+        data["full_org_type"] = user.full_org_type if user else None
+        data["organization_url"] = user.instance_url if user else None
```

Comment:
organization_url can be found in the ScratchOrg's config["instance_url"]. The others aren't as essential.

Reply:
Addressing this in #314